common/reviews/api/node-core-library.api.ts

```diff
@@ -416,6 +416,25 @@ class Sort {
   static sortSetBy<T>(set: Set<T>, keySelector: (element: T) => any, keyComparer?: (x: T, y: T) => number): void;
 }
 
+// @beta (undocumented)
+class StringBufferTerminalProvider implements ITerminalProvider {
```

Comment:
@iclanton since you created `StringBufferTerminalProvider` maybe you could help out with the docs here? For members that are simply implementing the `ITerminalProvider` contract, you can just put `/** {@inheritDoc ITerminalProvider.memberName} */` rather than copy+pasting the docs.

Reply:
Yeah I can fix the docs in a followup PR.